dialyzer.ignore-warnings

```diff
@@ -5,6 +5,8 @@ apps/omg_eth/test
 
 # TODO
 
+Unknown function 'Elixir.Sentry':capture_exception/2
```

Comment:
If you add `:sentry` in the PLT app list in `mix.exs` you don't need to add these ignore here.
But there's an open issues that will produce an umatched_return: https://github.com/getsentry/sentry-elixir/pull/344/files


Also, I think you forgot 
```
use Plug.ErrorHandler
use Sentry.Plug
```
to add in the integration.